tests/test_errors.py

```diff
@@ -125,6 +125,32 @@ def handle_custom_exception(error):
                 'test': 'value',
             })
 
+    def test_errorhandler_for_custom_exception_with_headers(self):
+        api = restplus.Api(self.app)
+
+        class CustomException(RuntimeError):
+            pass
+
+        @api.route('/test/', endpoint='test')
+        class TestResource(restplus.Resource):
+            def get(self):
+                raise CustomException('error')
+
+        @api.errorhandler(CustomException)
+        def handle_custom_exception(error):
+            return {'message': 'some maintenance', 'headers': {'Retry-After': 120}}, 503
```

Comment:
Using the same style as Resources methods response would be better because homogeneous:
```python
@api.errorhandler(CustomException)
def handle_custom_exception(error):
    return {'message': 'some maintenance'}, 503, {'Retry-After': 120}
```